### ISSUE TITLE
More visibility for the translation tool

### ISSUE THREAD

**Issue by Cwpute** [the-djmaze/snappymail#1353]
**Is your feature request related to a problem? Please describe.**
I wanted to know how to help translating Snappymail but couldn't find anything. I finally found a link in a issue #806, which didn't really explain what to do with it and how to do it.
Which tool is it anyway ? i've only ever worked with Weblate and Crowdin, and i'm not familiar with lightweight tools like this one.

**Describe the solution you'd like**
Perhaps a wiki page explaining how the translation process is executed, and easier ways to find the translation link.
Maybe feature the link to the translation tool on this github page's description, on the official website too, and perhaps in the Snappymail itself in an About page.

**Comment by the-djmaze** [comment]
Correct. The https://snappymail.eu/translate.php is just a simple script i wrote for the current languages.
It allows to see which translations are missing and you can modify others.

I don't use Weblate, Crowdin, Transifex, Google Translate, etc. because that would be yet another system with yet another login with yet another ...
While GitHub is fine to commit your changes to as `Pull request`.

The tool could be improved though

**Comment by Cwpute** [comment]
I can see this tool being used by people who are not familiar with git (like me)! Especially considering how lightweight it seems, it could be used in other projects.

**Comment by the-djmaze** [comment]
#1358
